packages/graphql-server-lambda/tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+  "compilerOptions": {
+    "target": "es6",
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "sourceMap": true,
+    "declaration": true,
+    "noImplicitAny": false,
+    "rootDir": "./src",
+    "outDir": "./dist",
+    "allowSyntheticDefaultImports": false,
+    "pretty": true,
+    "removeComments": true,
+    "typeRoots": [
+        "node_modules/@types"
+    ],
+    "types": [
+        "typed-graphql",
```

Comment:
please remember to remove (since we moved to @types/graphql) when rebasing this.